zcash_client_backend/src/data_api/error.rs

```diff
@@ -33,6 +35,9 @@ pub enum Error<DataSourceError, CommitmentTreeError, SelectionError, FeeError> {
     /// An error in note selection
     NoteSelection(SelectionError),
 
+    /// An error in change selection during transaction proposal construction
+    Change(ChangeError<ChangeErrT, NoteRefT>),
```

Comment:
The fact that this argument is of the form `crate::fees::ChangeError<ChangeErrT, NoteRefT>` does not appear to be used. Consider whether using just `ChangeError` (as a type parameter, not imported from `crate::fees`) would be sufficient.

Reply:
I don't understand this suggestion.

Reply:
Maybe Daira-Emma means the NoteRef side of it?

IIUC the error is actually used on line 130 and 238 and as associated type of the enum `InputSelectorError`. Although I'm not really well versed on this advanced usage of generics in RustLang.